prow/plugins/lgtm/lgtm.go

```diff
@@ -60,9 +60,17 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	configInfo := map[string]string{}
+	for _, orgRepo := range enabledRepos {
+		parts := strings.Split(orgRepo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", orgRepo)
+		}
+		configInfo[orgRepo] = fmt.Sprintf("The plugin has the following configuration: %+v.", optionsForRepo(config, parts[0], parts[1]))
```

Comment:
I think we need three sentences here, for `review_acts_as_lgtm` `store_tree_hash` and `trusted_team_for_sticky_lgtm`